src/core/connection.c

```diff
@@ -3412,19 +3412,17 @@ QuicConnRecvVerNeg(
     // requested version. If it does, we are supposed to ignore it. Cache the
     // first supported version.
     //
-    QuicTraceLogConnVerbose(
-        RecvVerNeg,
-        Connection,
-        "Received Version Negotation:");
+    QuicTraceLogVerbose(
+        PacketRxVersionNegotiation,
+        "[C][TX][-] VN");
```

Comment:
IMO, keeping `Version Negotiation` instead of `VN` would make logs easier to understand.

Reply:
The logs on the client receive now match the logs on the server send.

Reply:
But this should be `RX`

Reply:
I'd argue we should update the server side log to make it easier to understand too then, but I won't block the PR on that.

Reply:
We have a standard format for packet logging. Take a look at packet.c and frame.c. This was the one exception to that format.